client/app/controllers/index.js

```diff
@@ -16,22 +16,12 @@ export default Ember.Controller.extend({
             }));
         },
         addCollection () {
-
-            let coll = {
-                title: this.get('newCollectionTitle'),
-                tags: '',
-                description: '',
-                createdBy: 'Marcia Gonzales',
-                dateCreated:  new Date(),
-                dateUpdated: new Date(),
-                items: []
-            };
-            $.post('/api/collections', coll).then(result => {
```

Comment:
Did you intentionally leave the redirect out? That works too, just checking. 
In that case though the new one should appear at the top of the list.